server/medical-contact/collection.ts

```diff
@@ -0,0 +1,124 @@
+import type {HydratedDocument, Types} from 'mongoose';
+import type {MedicalContact} from './model';
+import MedicalContactModel from './model';
+import AccountCollection from '../account/collection';
+
+class MedicalContactCollection {
+  /**
+   * Add a new medical contact
+   *
+   * @param {string} ownerId - The id of the owner of the medical contact
+   * @param {string} title - The username of the user
+   * @param {string} first_name - The username of the user
+   * @param {string} last_name - The username of the user
+   * @param {string} hospital - The username of the user
+   * @param {string} specialty - The username of the user
+   * @param {string} phone_number - The username of the user
+   * @param {string} notes - The username of the user
+   * @return {Promise<HydratedDocument<MedicalContact>>} - The newly created medical contact
+   */
+  static async addOne(ownerId: Types.ObjectId | string, title: string, first_name: string, last_name: string, hospital: string, specialty: string, phone_number: string, notes: string): Promise<HydratedDocument<MedicalContact>> {
+    
+    const active = true; // Since medical contact was just created, it is active
+    const medicalContact = new MedicalContactModel({
+        ownerId,
+        active, 
+        title, 
+        first_name, 
+        last_name, 
+        hospital, 
+        specialty, 
+        phone_number, 
+        notes
+    });
+    await medicalContact.save(); // Saves user to MongoDB
+    return medicalContact.populate('ownerId');
```

Comment:
Do we really need owner id to be populated? Users can only ever see their own data, so I think this is redundant.